src/Tribe/Views/V2/Hooks.php

```diff
@@ -566,6 +567,35 @@ public function manage_sensitive_info( $post ) {
 		}
 	}
 
+	/**
+	 * Fires to add JSON LD to Single Event
+	 *
+	 * @since TBD
+	 *
+	 */
+	public function mange_single_json_ld() {
```

Comment:
Rename this method to `print_single_event_json_ld` and, proxy, using to the container, to a JSON-LD dedicated class.
Create the `Views/V2/Template/Json_Ld` class and move the logic of this method in there.
We'll eventually move the JSON-LD logic, all of it, for single and archives, out of V1 and it will end up in there.
That class will only have one method, for now, and that is fine.

We want to avoid making Hook another Main class.